soapui-testserver-api/src/test/resources/test-recipes/request-with-base64-attachment.json

```diff
@@ -0,0 +1,17 @@
+{
+  "testSteps": [
+    {
+      "type": "REST Request",
+      "method": "GET",
+      "URI": "http://google.com/",
+      "attachments" : [
+        {
+          "contentType" : "text/plain",
+          "name" : "A Name",
+          "contentId" : "An id",
+          "content": "Q29udGVudA=="
```

Comment:
If I remember this correctly, what I did was an automatic base64 encoding of the attachment and we talked about later doing as a property as you say now. But right now, there is no property, it is automatically put as base64 encoded. We should do a property.